packages/core/src/core/themes/night.scss

```
@@ -1,79 +1,84 @@
+@use 'sass:list';
 @use '../tokens' as castor;
 
-@mixin night() {
-  @include castor.tokens();
+// $type?: class | raw;
+@mixin night($type1: null, $type2: null) {
+  $class: list.index(($type1, $type2), 'class');
+  $raw: list.index(($type1, $type2), 'raw');
```

If you will not be refactoring these two theme mixins, then:

```suggestion
  $types: ($type1, $type2);
  $class: list.index($types, 'class');
  $raw: list.index($types, 'raw');
```

---

I probably will, just didn't have the time yet.
Can probably take another look today.
In the mean time I'll update to your suggestion.